packages/smart-wallet/test/supports.js

```diff
@@ -76,9 +76,11 @@ const makeFakeBridgeManager = () =>
           return E(handler).fromBridge(obj);
         },
         initHandler(newHandler) {
+          !handler || Fail`Handler already set`;
           handler = newHandler;
         },
         setHandler(newHandler) {
+          handler || Fail`Handler not set`;
```

Comment:
eslint giving an error. would it work to use the check style on line 70?
```
          if (!handler) {
            Fail`No handler for ${bridgeId}`;
          }
```

Reply:
Oh. I suppose that would have been good for consistency.
I just pushd a fixup with `!!handler || Fail...` though.